### PR TITLE
Hello result information!

### PR DESCRIPTION
**Pull request by raspbianlike**
## Description

Describe your pull request here

## Type

```
BREAKING CHANGE MY NIGGER
```

## Checklist

*Check the checkbox(es) that apply.*

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) guide.
- [x] `./build.sh` is passing with every build step and every test locally.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I tested it on **Linux** and on **Windows** and it's working as intended.

## Discussion

*If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...*

## Reviewers

@Marc3842h, ...

### CODE REVIEW

File: Titan/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -18,6 +18,6 @@
 // Titan uses SemVer (Microsoft doesn't prefer that and displays a warning), thats why we disable the compiler warnings.
 [assembly: AssemblyVersion("1.6.0")]
 [assembly: AssemblyFileVersion("1.6.0")]
-[assembly: AssemblyInformationalVersion("1.6.0-<%GitHash%>")] // Will be edited by Cake Build Script
+[assembly: AssemblyInformationalVersion("1.6.0-e7b1319")] // Will be edited by Cake Build Script
```
Comment (Marc3842h) [inline Titan/Properties/AssemblyInfo.cs]: Please revert this back to `<%GitHash%>`.

File: TitanTest/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -19,6 +19,6 @@
 // TitanTest uses SemVer (Microsoft doesn't prefer that and displays a warning), thats why we disable the compiler warnings.
 [assembly: AssemblyVersion("1.6.0")]
 [assembly: AssemblyFileVersion("1.6.0")]
-[assembly: AssemblyInformationalVersion("1.6.0-<%GitHash%>")] // Will be edited by Cake Build Script
+[assembly: AssemblyInformationalVersion("1.6.0-e7b1319")] // Will be edited by Cake Build Script
```
Comment (Marc3842h) [inline TitanTest/Properties/AssemblyInfo.cs]: Please revert this back to `<%GitHash%>`.

File: Titan/Managers/ThreadManager.cs [hunk 3]
```diff
@@ -229,7 +246,8 @@ public void StartCommend(TitanAccount account, CommendInfo info)
                     var timeSpent = DateTime.Now.Subtract(account.StartEpoch.ToDateTime());
 
                     _log.Error("Connection to account {Account} timed out. It was not possible to " +
-                               "commend the target after {Timespan} seconds.", account.JsonAccount.Username, timeSpent.Seconds);
+                               "commend the target after {Timespan} seconds.", account.JsonAccount.Username,
+                        timeSpent.Seconds);
```
Comment (Marc3842h) [inline Titan/Managers/ThreadManager.cs]: Please fix formatting here.

File: Titan/Managers/ThreadManager.cs [hunk 4]
```diff
@@ -302,7 +320,8 @@ public void StartMatchResolving(TitanAccount account, LiveGameInfo info)
                     var timeSpent = DateTime.Now.Subtract(account.StartEpoch.ToDateTime());
 
                     _log.Error("Connection to account {Account} timed out. It was not possible to resolve the Match " +
-                               "ID for the target after {Timespan} seconds.", account.JsonAccount.Username, timeSpent.Seconds);
+                               "ID for the target after {Timespan} seconds.", account.JsonAccount.Username,
+                        timeSpent.Seconds);
```
Comment (Marc3842h) [inline Titan/Managers/ThreadManager.cs]: Please fix formatting here.

Comment (raspbianlike) [reply Titan/Managers/ThreadManager.cs]: " // Will be edited by Cake Build Script "
Are you fucking gay or what?

Comment (Marc3842h) [reply Titan/Managers/ThreadManager.cs]: Yes, the value will be edited by the Cake Build Script but is supposted to be reverted back after the build process finished. It seems you cancelled the build process mid-way. 

Also, please follow the [Contributor Covenant Code of Conduct](https://github.com/Marc3842h/Titan/blob/master/CODE_OF_CONDUCT.md) before discussing and contributing to the Titan repository and the issues section. Inappropriate language may result in you losing your Collaborator permissions.

**Pull request merged**